_rules/heading-descriptive-b49b2e.md

```diff
@@ -25,15 +25,21 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any element with the [semantic role][] of heading that is either [visible][] or [included in the accessibility tree][].
+This rule applies to any element with the [semantic role][] of `heading` that is either [visible][] or [included in the accessibility tree][].
```

Comment:
Don't we want here something like "inheriting semantic heading" or such?

Reply:
Correct, but these PRs haven't been merged yet…

Reply:
It has now :)